packages/grid/rollup.x-data-grid-pro.config.js

```diff
@@ -32,24 +31,32 @@ export default [
       return new RegExp(`(${packageName}|${packageName}\\/.*)`);
     }),
     plugins: [
-      replace({
-        __RELEASE_INFO__: generateReleaseInfo(),
-      }),
       production &&
         cleaner({
           targets: ['./x-data-grid-pro/build/'],
         }),
       typescript({ tsconfig: 'tsconfig.build.json' }),
       babel({
         babelHelpers: 'bundled',
-        extensions: ['.tsx'],
+        extensions: ['.tsx', '.ts'],
         plugins: [
           [
             'transform-react-remove-prop-types',
             {
               ignoreFilenames: ['DataGridPro.tsx'],
             },
           ],
+          [
+            'search-and-replace',
```

Comment:
I moved this replace to babel because once we remove rollup for `@mui/x-data-grid-pro` we won't be able to use to current plugin.